config/prometheus/prometheus.yaml

```diff
@@ -0,0 +1,9 @@
+global:
+  scrape_interval: 15s
+
+scrape_configs:
+  - job_name: 'odp-connect-stats'
+    scrape_interval: 5s
```

Comment:
This line is probably not needed. `5s` is unnecessary. The default `15s` should be fine.